pkg/virt-handler/hotplug-disk/mount.go

```diff
@@ -117,11 +117,11 @@ type vmiMountTargetRecord struct {
 }
 
 // NewVolumeMounter creates a new VolumeMounter
-func NewVolumeMounter(mountStateDir string) VolumeMounter {
+func NewVolumeMounter(mountStateDir string, kubeletPodsDir string) VolumeMounter {
 	return &volumeMounter{
 		mountRecords:       make(map[types.UID]*vmiMountTargetRecord),
 		mountStateDir:      mountStateDir,
-		hotplugDiskManager: hotplugdisk.NewHotplugDiskManager(),
+		hotplugDiskManager: hotplugdisk.NewHotplugDiskManager(kubeletPodsDir),
```

Comment:
You could use the already existing function [NewHotplugDiskWithOptions](https://github.com/kubevirt/kubevirt/blob/1a8f08103e48c5f2bb2f5826d118507ce7ec1f0c/pkg/hotplug-disk/hotplug-disk.go#L59)

Reply:
I think that there is no need to divide into two options 'NewHotplugDiskManager' and 'NewHotplugDiskWithOptions'.
Just need to forward the value 'KubeletPodsDir' of virt-handler... what do yo say ?

Reply:
Well, you are changing the function signature where there is a second function already existing that you could use to achieve exactly the same

Reply:
+1 for removing the function and pass the path